reaktive/src/commonMain/kotlin/com/badoo/reaktive/maybe/ZipWith.kt

```diff
@@ -0,0 +1,7 @@
+package com.badoo.reaktive.maybe
+
+fun <T, R> Maybe<T>.zipWith(other: Maybe<R>): Maybe<Pair<T, R>> =
+    zip(this, other) { first, second -> first to second }
+
+fun <T, R, I> Maybe<T>.zipWith(other: Maybe<R>, mapper: (T, R) -> I): Maybe<I> =
```

Comment:
mapper should be changed to zipper

Reply:
`zip(a, b, mapper)` has the lambdas named as `mapper` actually, `combineLatest` as well. This is small but maybe it's better to keep consistency?

Reply:
Sure